packages/react-charting/src/components/HorizontalBarChart/HorizontalBarChart.base.tsx

```diff
@@ -267,6 +267,8 @@ export class HorizontalBarChartBase extends React.Component<IHorizontalBarChartP
               : undefined
           }
           aria-labelledby={this._calloutId}
+          role="img"
+          aria-label="Horizontal bar chart"
```

Comment:
We should take a props for this. Hard coding this will cause it be not localized